package.json

```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "eslint": "3.7.1",
```

Comment:
it's ok, just make sure you are fully concentrated next time. It's important to stay focused when pressing buttons!